modules/msk/mutect1/tests/main.nf.test

```diff
@@ -0,0 +1,49 @@
+nextflow_process {
+
+    name "Test Process MUTECT1"
+    script "../main.nf"
+    process "MUTECT1"
+
```

Comment:
I'd include the nextflow.config here.

Reply:
a36ee1214bb9a15bfa6797c356d7dc7263f351dc